operator/controller/src/test/java/io/apicurio/registry/operator/it/KafkaSQLITTest.java

```diff
@@ -0,0 +1,104 @@
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+import static io.apicurio.registry.operator.resource.ResourceFactory.deserialize;
+import static io.apicurio.registry.operator.resource.ResourceFactory.load;
+import static org.assertj.core.api.Assertions.as;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+@QuarkusTest
+public class KafkaSQLITTest extends ITBase {
+
+    private static final Logger log = LoggerFactory.getLogger(KafkaSQLITTest.class);
+
+    @BeforeAll
+    public static void beforeAll() throws Exception {
+        ITBase.beforeAll();
+        applyStrimziResources(false);
+    }
+
+    @Test
+    void testKafkaSQLPlain() {
+        var kafka = deserialize("/k8s/examples/kafkasql/plain/ephemeral.kafka.yaml", Kafka.class);
+        client.resource(kafka).create();
+
+        await().ignoreExceptions().until(() -> {
+            var status = client.resources(Kafka.class).inNamespace(namespace).withName("my-cluster").get()
+                    .getStatus();
+            assertThat(status.getConditions()).filteredOn(c -> "Ready".equals(c.getType())).singleElement()
+                    .extracting("status", as(InstanceOfAssertFactories.STRING)).isEqualTo("True");
+            return true;
+        });
+
+        // get plain bootstrap servers
+        var status = client.resources(Kafka.class).inNamespace(namespace).withName("my-cluster").get()
+                .getStatus();
+        var bootstrapServers = status.getListeners().stream().filter(l -> "plain".equals(l.getName()))
+                .map(ListenerStatus::getBootstrapServers).findFirst().get();
+
+        var registry = deserialize("k8s/examples/kafkasql/plain/kafka-plain.apicurioregistry3.yaml",
+                ApicurioRegistry3.class);
+        registry.getMetadata().setNamespace(namespace);
+        registry.getSpec().getApp().getKafkasql().setBootstrapServers(bootstrapServers);
+
+        client.resource(registry).create();
+
+        await().ignoreExceptions().until(() -> {
+            assertThat(client.apps().deployments().inNamespace(namespace)
+                    .withName(registry.getMetadata().getName() + "-app-deployment").get().getStatus()
+                    .getReadyReplicas().intValue()).isEqualTo(1);
+            var podName = client.pods().inNamespace(namespace).list().getItems().stream()
+                    .map(pod -> pod.getMetadata().getName())
+                    .filter(podN -> podN.startsWith(registry.getMetadata().getName() + "-app-deployment"))
+                    .findFirst().get();
+            assertThat(client.pods().inNamespace(namespace).withName(podName).getLog())
+                    .contains("Using Kafka-SQL artifactStore");
+
+            return true;
+        });
+    }
+
+    @AfterAll
+    public static void afterAll() throws Exception {
+        applyStrimziResources(true);
```

Comment:
Reverted. I consider this fine, since it is often easier to fix a small thing immediately if you notice it, rather than trying to remember and create micro PRs. Do you consider it a good practice to track things like these in issues? If so I'll create an issue for this.